spec/tests/duration.fmf

```diff
@@ -12,8 +12,12 @@ description: |
     First, all time values are summed together, and only then are they multiplied.
     The final value is then rounded up to the whole number.
 
+    Note that the asteric has a special meaning in YAML syntax and thus you need to
```

Comment:
Typo, it's "asterisk". I'd also add "... asterisc character, "*", ..."

Reply:
good point. Fixed and force pushed to 0f5fbcc0